code/datums/status_effects/debuffs/debuffs.dm

```diff
@@ -178,31 +178,68 @@
 	ADD_TRAIT(owner, TRAIT_KNOCKEDOUT, TRAIT_STATUS_EFFECT(id))
 	tick_interval = initial(tick_interval)
 
+#define HEALING_SLEEP_DEFAULT -0.2
+
 /datum/status_effect/incapacitating/sleeping/tick()
 	if(owner.maxHealth)
 		var/health_ratio = owner.health / owner.maxHealth
-		var/healing = -0.2
+		var/healing = HEALING_SLEEP_DEFAULT
```

Comment:
Flip the signs of all of this, `healing` should not be a negative number if you're regaining health

Reply:
```dm
owner.adjustBruteLoss(healing)
owner.adjustFireLoss(healing)
```
These procs are how healing is used.  Adding a positive number is damage and negative number is healing damage.

Reply:
healing can be positive and then you do -healing on the proc itself!

Reply:
Applied with 112d67996c357d2d0c1f824ef1140154b6cc403f